mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -329,8 +329,7 @@ def teardown(self) -> None:
         single call.
         """
         _LOG.info("Teardown %s", self)
-        # TODO: Make sure we create a context before invoking setup/run/status/teardown
```

Comment:
Maybe leave this comment and just remove the TODO?  It helps explain the assert a little bit.

Reply:
makes perfect sense! will do it now